run.py

```diff
@@ -154,6 +155,8 @@
                                     rhel images for Interop runs
   --skip-sos-report                 Enables to collect sos-report on test suite failures
                                     [default: false]
+  --collect-coredump                Enables presevation of coredump directory
+                                    [default: false]
```

Comment:
IMHO, I would like to avoid providing such nobs... unless it is required by multiple modules, then it makes sense.

If it is required only in the `install_prereq` see if you could leverage `--custom-config install_prereq_enable_coredump`